core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala

```diff
@@ -19,6 +21,9 @@ object LogOf {
 
   def summon[F[_]](implicit F: LogOf[F]): LogOf[F] = F
 
+  def logger[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
```

Comment:
could we cal it `log` after the class `Log` ?

Reply:
done